.cirrus.yml

```diff
@@ -476,12 +494,13 @@ special_testing_in_podman_task:
         $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:IMG.*' &&
         $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:DOCS.*'
 
-    gce_instance:
-        matrix:
-            # FIXME: Integration testing currently broken for F31 hosts
-            # Error: container_linux.go:345: starting container process caused "process_linux.go:281: applying cgroup configuration for process caused \"mountpoint for cgroup not found\"": OCI runtime error
-            # image_name: "${FEDORA_CACHE_IMAGE_NAME}"
-            image_name: "${PRIOR_FEDORA_CACHE_IMAGE_NAME}"
+    matrix:
+        # FIXME: Integration testing currently broken for F31 hosts
+        # Error: container_linux.go:345: starting container process caused "process_linux.go:281: applying cgroup configuration for process caused \"mountpoint for cgroup not found\"": OCI runtime error
+        # image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+        - name: "special_testing_in_podman ${PRIOR_FEDORA_NAME}"
```

Comment:
Is it necessary to have the task-level `alias: "special_testing_in_podman"` also if you're setting `name` here?  Might be confusing to future maintainers as it's a bit ambiguous as to which name will be used/presented and where.  Though I can understand if this isn't fixable.

Reply:
This is horrible but yes, it's necessary, because otherwise the only thing appearing in name is "fedora-30" with no way to distinguish it from other tasks. I believe the horribleness tradeoff is worth it for improved end-user experience.

Reply:
For sure it is, but since it's horrible and required, it's probably worth documenting or commenting somewhere.

Idea (not sure if possible / would work): Maybe a YAML anchor & alias could help avoid value duplication?   i.e. Anchor on `alias: "special_testing_in_podman" &something` and (somehow) alias that into `name: *something "${PRIOR_FEDORA_NAME}"` (not sure that can be done with scalars).